src/composition/auth.ts

```diff
@@ -24,7 +24,7 @@ const openLogin = (source: string) => {
   return window.open(url, '_blank');
 };
 
-const loginSourceList = MediaService.getLoginProviders().map((i) => i.name);
+const loginSourceList = MediaService.getLoginProviders().map((i) => i._name);
```

Comment:
if name can be accessed outside, it's better to rename it to name without underscore

Reply:
Static property 'name' conflicts with built-in property 'Function.name'

Reply:
maybe add `getName()` with `_name` property?

Reply:
if _name is reserved for internal use. What about `Name`?